qa/rpc-tests/test_framework/script.py

```diff
@@ -17,7 +17,18 @@
 
 from .mininode import CTransaction, CTxOut, sha256, hash256, uint256_from_str, ser_uint256, ser_string
 from binascii import hexlify
-import hashlib
+from .bignum import bn2vch
+from .ripemd160 import ripemd160
+
+import sys
+bchr = chr
+bord = ord
+if sys.version > '3':
+    long = int
+    bchr = lambda x: bytes([x])
+    bord = lambda x: x
+
+import struct
```

Comment:
Additional finding, sorry for fragmented review - I forgot to write this:

I think this simply can be:

```suggestion
from .ripemd160 import ripemd160
````

Reply:
Fixed but will let next reviewer mark as resolved. Thanks!